### PR TITLE
feat: release 0.40.0

### PR DESCRIPTION
**Pull request by magnus-madsen**

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+Version 0.40.0:
+
+- Library: Added `Adaptor` module and `ToJava` class for working with Java collections (thanks Stephen!)
```
Comment (mlutze) [inline docs/CHANGELOG.md:5]: Do we want to publish this, given they depend on an unstable feature?

Comment (magnus-madsen) [reply docs/CHANGELOG.md:5]: Rest OK and spell check?

File: docs/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+Version 0.40.0:
+
+- Library: Added `MutPriorityQueue` (thanks Xavier!)
+- VSCode: Added completion for `use` of signatures and operations (thanks Xavier!)
+- VSCode: Added code actions to derive missing type class instances (thanks Holger!)
+- VSCode: Added code actions for unused code (thanks Holger!)
+- Compiler: Added new HTML API Documentation tool (thanks Holger!)
```
Comment (mlutze) [inline docs/CHANGELOG.md:9]: I'd lowercase `documentation` :)

Comment (mlutze) [reply docs/CHANGELOG.md:9]: Otherwise looks good

**Pull request merged**